client/src/document/index.scss

```diff
@@ -22,13 +22,23 @@
   }
 }
 
+/* override rule from reset as it does not work for MDN */
+article > * + * {
+  margin-top: 0;
+}
+
 .article {
-  padding: $base-spacing;
+  padding: ($base-spacing / 2) $base-spacing $base-spacing;
 
   @media #{$mq-small-desktop-and-up} {
     margin-bottom: $large-spacing;
   }
 
+  /* if the article starts with a notecard, remove the top margin */
```

Comment:
Why can't we just remove ALL top margins from all elements? 
I'd rather not land this hack. Especially since there's no plan for how to fix it post-launch. If there was a filed issue we could maybe write something like `/* Because so and so... This is a temporary HACK until we address https://github.com/issues/1234 */`